pkg/api/common/versions.go

```diff
@@ -351,6 +351,13 @@ func IsKubernetesVersionGe(actualVersion, version string) bool {
 	return v1.GE(v2)
 }
 
+// IsKubernetesVersionEq returns true if actualVersion is equal to version
+func IsKubernetesVersionEq(actualVersion, version string) bool {
```

Comment:
why not just compare the strings for this?

If you decide to keep it, it'd be an easy unit test to add in versions_test.go

Reply:
Originally I was hoping the semantic versioning library could handle some cases like "1.11" == "1.11.0" . It doesn't handle that case so I should probably keep it simple and remove it.